plutus-core/cost-model/data/builtinCostModel.json

```diff
@@ -18,7 +18,7 @@
             "type": "constant_cost"
         },
         "cpu": {
-            "arguments": 2.71491123389651,
+            "arguments": 5.43034381132275,
```

Comment:
God question, but I don't know.  The numbers in `benching.csv` definitely  back that up.  There are quite a lot of discrepancies between the old and new versions of `benching.csv`.  I think there have been quite a lot of changes in the rest of the code since the previous version of the csv file was created, but I've lost track of exactly what's changed.  I'lll run the benchmarks again soon just to make sure the output is similar.

Reply:
Good question, but I don't know the answer. The figures in `benching.csv` definitely back that up. I think the previous version of `benching.csv` is quite old and I've lost track of what else has changed since it was created (I merged this branch with master quite recently to bring it up to date).  I'll run the benchmarks again later to check that the results are similar.